components/viaduct/examples/example_ohttp_flow.rs

```diff
@@ -0,0 +1,71 @@
+#[cfg(all(feature = "ohttp", feature = "backend-dev"))]
```

Comment:
This example code is great, but it's going to need some changes since I've been moving things around in preparation of the monorepo work:

- Let's move it into `viaduct-cli`, I think it would work as a subcommand there.  Moving it to `viaduct-cli` affects how moz-central treats the dependencies.  They won't get vendored in and we only need to vet them as `safe-to-run` rather than `safe-to-deploy`, which is a lesser level.
- Remove the `env_logger` dependency from `viaduct`.
- Remove the references to the deleted `viaduct/backend-dev` feature.

Also, let's use the newer, tracing-based logging by:
- Add the `error-support/tracing-logging` feature to `viaduct`.
- Add a `cli_support::init_logging_with` line in the viaduct CLI code, see the other CLIs for examples.